detection-rules/link_google_amp_suspicious_indicators.yml

```
@@ -0,0 +1,56 @@
+name: "Credential Phishing: Google Accelerated Mobile Pages (AMP) abuse"
+description: |
+  This rule is designed to identify phishing attempts exploiting Google AMP's URL structure for malicious activities. 
+  The rule aims to detect specific URL patterns, further analyzing both message content, as well as the destination of the link to distinguish
+  between legitimate Google AMP pages and potential malicious usage.
+references:
+  - "https://cofense.com/blog/google-amp-the-newest-of-evasive-phishing-tactic/"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  
+  // Any body links with a domain SLD of 'google' and a path starting with /amp/s
+  and any(body.links,
+          .href_url.domain.sld == "google"
+          and strings.starts_with(.href_url.path, "/amp/s/")
+  
+          // Brand Logo detected that is not google
+          and (
+            any(ml.logo_detect(beta.message_screenshot()).brands,
+                .name is not null and .name != "Google"
+            )
+            
+            // or credphishin login or captcha
```

```suggestion
            // or the page has a login or captcha
```
for clarity, this is what this is really doing -- doesn't have to be credphish'ing related at all, that's just where the output is returned